atomicapp/providers/docker.py

```diff
@@ -105,7 +106,10 @@ def run(self):
             if self.dryrun:
                 logger.info("DRY-RUN: %s", " ".join(cmd))
             else:
-                subprocess.check_output(cmd)
+                try:
+                    subprocess.check_output(cmd)
+                except subprocess.CalledProcessError as e:
+                    raise DockerException("DRY-RUN: Failed! %s. \n%s" % (cmd, e.output))
```

Comment:
This message is wrong.. Should not contain the work DRY-RUN because this is the "non dry-run" case.